robozonky-strategy-natural/src/main/antlr4/imports/CommonFilters.g4

```diff
@@ -99,6 +99,52 @@ relativeTermConditionRangeClosedRight returns [RelativeLoanTermCondition result]
     { $result = new RelativeLoanTermCondition(0, $max.result - 1); }
 ;
 
+elapsedTermCondition returns [ElapsedLoanTermCondition result]:
+    'uhrazeno ' (
+        (c1 = elapsedTermConditionRangeOpen { $result = $c1.result; })
+        | (c2 = elapsedTermConditionRangeClosedLeft { $result = $c2.result; })
+        | (c3 = elapsedTermConditionRangeClosedRight { $result = $c3.result; })
+    ) ' splátek'
+;
+
+elapsedTermConditionRangeOpen returns [ElapsedLoanTermCondition result]:
+    IS min=intExpr UP_TO max=intExpr
+    { $result = new ElapsedLoanTermCondition($min.result, $max.result); }
+;
+
+elapsedTermConditionRangeClosedLeft returns [ElapsedLoanTermCondition result]:
+    'více než' min=intExpr
```

Comment:
Any reason not to use MORE_THAN (as opposed to literal string)?

Reply:
Exactly. There was no way to make a decent Czech sentence using the standard MORE_THAN.